src/olympia/amo/views.py

```diff
@@ -93,7 +93,7 @@ def handler404(request):
 
 @non_atomic_requests
 def handler500(request):
-    if request.is_legacy_api:
+    if getattr(request, 'is_legacy_api', False):
```

Comment:
Why is this necessary?

Reply:
It probably isn't in the end - there's an exception thrown, seemingly before the `is_legacy_api` property is added to `request`, and without this the real exception is replaced with one about `is_legacy_api` being undefined..

Reply:
ok, it kinda makes sense anyway...